project-base/app/src/FrontendApi/Resolver/Image/Exception/ImageTypeInvalidUserError.php

```diff
@@ -1,21 +0,0 @@
-<?php
-
-declare(strict_types=1);
-
-namespace App\FrontendApi\Resolver\Image\Exception;
-
-use Shopsys\FrontendApiBundle\Model\Error\EntityNotFoundUserError;
-use Shopsys\FrontendApiBundle\Model\Error\UserErrorWithCodeInterface;
-
-class ImageTypeInvalidUserError extends EntityNotFoundUserError implements UserErrorWithCodeInterface
```

Comment:
it seems like `ImageTypeInvalidUserError` is not used anywhere, could you please check it and remove it as well if possible?

Reply:
removed 23336c3113